### ISSUE TITLE
HA VMs restarted.ps1 not reporting to the check list

### ISSUE THREAD

**Issue by Manikmp23** [alanrenouf/vCheck-vSphere#710]
Hi , Any body help for , while running vcheck script , HA VMs restarted.ps1 not reporting to the check list

**Comment by lucdekens** [comment]
Did you check that your vCenter is retaining the events sufficiently long?

**Comment by Manikmp23** [comment]
yes we have checked events are sufficiently long ( how ever we tried with recent logs

**Comment by lucdekens** [comment]
Can you check if there VM restarted events present in your vCenter?
```
Get-VIEvent -start (Get-Date).adddays(-5) -MaxSamples ([int]::MaxValue) | 
Where-Object {$_.EventTypeId -eq "com.vmware.vc.ha.VmRestartedByHAEvent"} | 
Select ObjectName,CreatedTime
```

**Comment by Manikmp23** [comment]
yes it is working but how ever it will not replicate in health check report / (Version 6.7.0.42000 Build 15976668 - we have recently upgraded our webclient)  please suggest
